org-roam-capture.el

```diff
@@ -303,6 +295,21 @@ This function is used solely in Org-roam's capture templates: see
         (push val converted)))
     (append (nreverse converted) `(:org-roam ,org-roam-plist))))
 
+(defun org-roam-capture--find-file-h ()
+  "Run the hooks defined in `org-roam-capture-after-finalize-hook'.
```

Comment:
the docstring is incorrect now.

Reply:
On it.

Reply:
Fixed in a6ac2b58961efbfdee7479f16bbbd1201ca4f9a8.